web/src/components/SingleRoom.tsx

```diff
@@ -40,7 +40,7 @@ const SingleRoom: React.FC<Props> = ({ roomId, userId }) => {
           <a href={window.location.href}>Link to this room</a> (Share this with
           your colleagues)
         </div>
-        <div>
+        <div style={{ color: "red" }}>
```

Comment:
We use "css" at the moment. Please write styles in `SingleRoom.css` and specify `className="SingleRoom-foo"` here. (change "foo")